src/test/java/ru/practicum/shareit/booking/BookingServiceImplTest.java

```diff
@@ -3,83 +3,590 @@
+        Booking booking = Booking.builder()
+                .id(bookingId)
+                .start(DEFAULT_START_DATE)
+                .end(DEFAULT_END_DATE)
+                .status(BookingStatus.WAITING)
+                .item(item1)
+                .booker(user2)
+                .build();
+        Mockito.when(userStorage.existsById(anyLong())).thenReturn(true);
+        Mockito.when(bookingStorage.findByItem_OwnerIdAndStatusOrderByStartDesc(anyLong(), any(), any()))
+                .thenReturn(List.of(booking));
+        //when
+        List<BookingResponseDto> list = bookingService.getListByOwner(2L, BookingStatus.WAITING, 0L, 20);
+        //then
+        assertEquals(response1Dto.getStart(), list.get(0).getStart());
+        assertEquals(response1Dto.getEnd(), list.get(0).getEnd());
+        assertEquals(response1Dto.getId(), list.get(0).getId());
+        assertEquals(response1Dto.getStatus(), list.get(0).getStatus());
+        assertEquals(response1Dto.getItem().getName(), list.get(0).getItem().getName());
+        assertEquals(response1Dto.getBooker().getName(), list.get(0).getBooker().getName());
+    }
+
+    @Test
+    void deleteById_whenInputOk_thenOk() {
+        //given
+        Mockito.when(bookingStorage.findById(any())).thenReturn(Optional.of(booking1ByUser2));
+        //when
+        String message = bookingService.deleteById(user2.getId(), booking1ByUser2.getId());
+        //then
+        assertEquals(Constants.SUCCESS_DELETE_MESSAGE, message);
+        Mockito.verify(bookingStorage).deleteById(booking1ByUser2.getId());
+    }
+
+    @Test
+    void deleteById_whenUserNotBookingOwner_thenThrowsForbiddenException() {
+        //given
+        Long user1IdNotBookerOwner = 1L;
+        Mockito.when(bookingStorage.findById(any())).thenReturn(Optional.of(booking1ByUser2));
+        //when
+        ForbiddenException fe = assertThrows(ForbiddenException.class,
+                () -> bookingService.deleteById(user1IdNotBookerOwner, booking1ByUser2.getId())
+        );
+        //then
+        assertEquals(format("User with id %d is not related to booking", user1IdNotBookerOwner), fe.getMessage());
+        Mockito.verify(userStorage, never()).deleteById(any());
+    }
+
+    /**
+     * вспомогательный метод настройки сущностей для теста
+     */
+    private void setupUsersAndItems() {
+        user2 = User.builder().id(2L).name("user2").email("user2@host.dom").build();
+        item1 = Item.builder().id(1L).ownerId(1L).name("item1").description("description1").available(true).build();
+        item2 = Item.builder().id(2L).ownerId(2L).name("item2").description("description2").available(true).build();
+    }
+
+    /**
+     * вспомогательный метод настройки dto для теста
+     * @param start время начала бронирования
+     * @param end время конца бронирования
+     */
+    private void setupEntityDtos(LocalDateTime start, LocalDateTime end) {
+        booking1Dto = BookingDto.builder()
+                .start(start)
```

Comment:
Соответственно здесь и ниже будет
```suggestion
                .start(BookingServiceImplTest.DEFAULT_START_DATE)
```

Reply:
да, действительно...спасибо